src/stc/PlatWX.cpp

```diff
@@ -295,15 +297,15 @@ void SurfaceImpl::InitPixMap(int width, int height, Surface *surface, WindowID w
         hdc = new wxMemoryDC(static_cast<SurfaceImpl*>(surface)->hdc);
     else
         hdc = new wxMemoryDC();
+    ((wxMemoryDC*)hdc)->GetImpl()->SetWindow(GETWIN(winid));
```

Comment:
I'd add `wxMemory* mdc`, initialize it appropriately and then do `hdc = mdc` to avoid the ugly cast.